spec/requests/users_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
```

Comment:
Has `users_spec.rb` and `posts_spec.rb` in `spec/requests/` directory 😎 Awesome!